core/templates/services/context.service.ts

```diff
@@ -40,16 +40,15 @@ export class ContextService {
   // NOTE TO DEV: Make sure any tests that directly access these variables clear
   // it (using the appropriate reset fn) initially. Since these are static,
   // depending on the order of tests, values may be retained across tests.
-  static customEntityContext = null;
+  static customEntityContext: EntityContext | null = null;
   static imageSaveDestination: string = (
     AppConstants.IMAGE_SAVE_DESTINATION_SERVER);
 
-  pageContext = null;
-  explorationId = null;
-  explorationIsLinkedToStory = false;
-  questionPlayerIsManuallySet = false;
-  questionId = null;
-  editorContext = null;
+  pageContext: string | null = null;
```

Comment:
Maybe add a comment here explaining why the properties can be `null`?

Reply:
done